Source/WebKit/UIProcess/Extensions/WebExtension.cpp

```diff
@@ -245,6 +247,68 @@ void WebExtension::populateWebAccessibleResourcesIfNeeded()
         parseWebAccessibleResourcesVersion2();
 }
 
+URL WebExtension::resourceFileURLForPath(const String& originalPath)
+{
+    ASSERT(originalPath);
+
+    String path = originalPath;
+
+    if (path.startsWith('/'))
+        path = path.substring(1);
+
+    if (!path.length() || m_resourceBaseURL.isNull())
+        return { };
+
+#if PLATFORM(COCOA)
+    auto resourceURL = static_cast<NSURL *>(URL(m_resourceBaseURL, path)).URLByStandardizingPath;
```

Comment:
This will need to be in the platform specific file as it is ObjC.

Also `resourceURL` should be renamed to `resourcePath`.